packages/iris-grid/src/sidebar/CustomColumnBuilder.tsx

```diff
@@ -336,15 +336,11 @@ class CustomColumnBuilder extends Component<
     const saveText = inputs.length > 1 ? 'Save Columns' : 'Save Column';
 
     return (
-      <button
-        type="button"
-        className={classNames(
-          'btn btn-apply',
-          {
-            'btn-spinner': isCustomColumnApplying,
-          },
-          isSuccessShowing ? 'btn-success' : 'btn-primary'
-        )}
+      <Button
```

Comment:
Not part of your change, but I noticed the padding is incorrect on the title "Custom Column Name and Formula":
![image](https://user-images.githubusercontent.com/4505624/196716911-a7164029-1016-45b9-8198-3c8856f5a236.png)

If you can fix this as part of this change, that would be helpful. Check with @dsmmcken if we want to just add the correct padding there (so it lines up with the inputs below), or if we can just remove that subtitle (it seems redundant)

Reply:
The subtitle will be removed.

Reply:
That padding is always broken for me, but only in dev mode. Works fine in a built version. Not sure why it's broken in dev though